frontend/cypress/e2e/login.cy.js

```diff
@@ -30,6 +26,11 @@ describe('Failing or Succeeding to Login', function () {
                     cy.get('div[role=\'status\']').should('be.visible');
                 }
             });
+            cy.wait(2000);
+            cy.url().should('include','/');
+            cy.get('[data-test="panelSwitchIcon"').should('be.visible');
+            cy.get('[data-test="openMenu"').should('be.visible');
+            cy.get('[data-test="logo"').should('be.visible').should('have.id','header-logo');
         });
```

Comment:
Add another step for a successfull login atleast to ensure login part is done. Then this should be good to be merged.

Reply:
@sherrif10 login has already done i just refractor and will add more to it